status_dashboard.py

```diff
@@ -0,0 +1,97 @@
+from retriever import install_sqlite
+from retriever.lib.engine_tools import getmd5
+from retriever import datasets
+from retriever.engines import engine_list
+from tempfile import mkdtemp
+from shutil import rmtree
+import os
+import urllib.request
+import json
+from datetime import datetime
+
+mysql_engine, postgres_engine, sqlite_engine, msaccess_engine, csv_engine, download_engine, json_engine, xml_engine = engine_list
+
+file_location = os.path.dirname(os.path.realpath(__file__))
+
+example_datasets = ['bird-size', 'mammal-masses', 'airports', 'portal']
+precalculated_md5 = {'airports': 'f16a0bb21d347e2ba180cde26f35c87a',
```

Comment:
Do we need these hash numbers if you already created a separate file for that kind of data?

Reply:
@henrykironde No we don't need it now.